<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenimage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="quality_label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality %</string>
    </property>
   </widget>
   <widget class="QSlider" name="percent">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_size_image1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>630</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>size_image 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_size_image2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>630</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>size_image 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
